controller/src/test/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessorsTest.java

```diff
@@ -90,13 +91,54 @@ public void testEventKey() {
         assertEquals(commitEvent.getKey(), "test/test");
     }
 
+    @Test(timeout = 10000)
+    public void testIsReady() {
+        Controller localController = mock(Controller.class);
```

Comment:
Most of these objects should be closed, right? If not, this test is going to leak resources. Please, add `@Cleanup` on all the objects that require it.

Reply:
These are mock objects and not necessarily need `@cleanup`. The same behavior is observed across `Controller` tests (not sure about SSS). In fact in this class itself.

Reply:
Ok, right, missed the `mock` part.